### ISSUE TITLE
Error: Cannot write (on export)

### ISSUE THREAD

**Issue by piotr-yuxuan** [flexsurfer/re-frisk#30]
Sorry for this rather scarce error report, I don't know where to gather further data.

When I click on "export" button I get this error
<img width="571" alt="screen shot 2017-02-21 at 00 42 54" src="https://cloud.githubusercontent.com/assets/3686502/23145337/e6040756-f7cd-11e6-938f-0816c7b7f3c5.png">

I've tried on Firefox 53.0a2 (2017-02-02) (64-bit) and Chrome Version 58.0.3011.0 (Official Build) canary (64-bit). I use `[re-frisk "0.3.2"]`, `[re-frame "0.9.1"]`, `[org.clojure/clojurescript "1.9.293"]` and (ifneedsbe) `[org.clojure/clojure "1.8.0"]`.

My platform is:
```
$ uname -a # (macOS)
Darwin … 16.3.0 Darwin Kernel Version 16.3.0
```

**Comment by flexsurfer** [comment]
hi Пëтр, it seems you have data in you app-db which cannot be mapped https://github.com/cognitect/transit-cljs#default-type-mapping

**Comment by flexsurfer** [comment]
fixed in 0.4.4

**Comment by piotr-yuxuan** [comment]
Really fixed? wow it's amazing!
